services/oc-opencor-base/.cookiecutterrc

```diff
@@ -31,7 +31,7 @@ default_context:
     project_git_repo:          'https://github.com/ITISFoundation/opencor'
     project_name:              'opencor'
     project_package_name:      'opencor'
-    project_short_description: 'OpenCOR is a cross-platform modelling environment, which can be used to organise, edit, simulate and analy'
+    project_short_description: 'OpenCOR is a cross-platform modelling environment, which can be used to organise, edit, simulate and analyze CellML and SED-ML files'
```

Comment:
Mixing American and British English ("analyze" rather than "analyse"). British English is used elsewhere, so might want to stick to it.

Reply:
ok